jest.config.js

```diff
@@ -1,35 +0,0 @@
-/* https://jestjs.io/docs/configuration */
-
-export default {
-  coverageProvider: 'v8',
```

Comment:
`v8` is the default [coverage provider](https://vitest.dev/config/#coverage-provider).